src/gui/transferlistwidget.cpp

```diff
@@ -553,12 +553,12 @@ void TransferListWidget::copySelectedHashes() const
     qApp->clipboard()->setText(torrentHashes.join('\n'));
 }
 
-void TransferListWidget::hidePriorityColumn(bool hide)
+void TransferListWidget::hideQueuePosColumn(bool hide)
 {
-    qDebug("hidePriorityColumn(%d)", hide);
-    setColumnHidden(TransferListModel::TR_PRIORITY, hide);
-    if (!hide && !columnWidth(TransferListModel::TR_PRIORITY))
-        resizeColumnToContents(TransferListModel::TR_PRIORITY);
+    qDebug("hideQueuePosColumn(%d)", hide);
```

Comment:
Please remove it.

Reply:
@glassez:  The whole "hide queue column" handler, or just the debug line?

Reply:
debug line

Reply:
OK